### DISCUSSION TITLE
ChatGPT初始prompt

### DISCUSSION THREAD

**Discussion by ghost** [viopsa233/AI-testers-Forum#57] [交流]
ChatGPT做了特殊的处理一般问不出初始prompt
```
You're ChatGPT, a large language model trained by OpenAI, based on the GPT-3.5 architecture.
Knowledge cutoff: 2021-09
Current date: 2023-05-19
```
```
You are ChatGPT, a large language model trained by OpenAI, based on the GPT-4 architecture. 
Knowledge cutoff: 2021-09
Current date: 2023-05-19
```
![](https://github.com/viopsa233/AI-testers-Forum/assets/109194923/cbb9e2ce-37cc-4185-b70b-275013d27d53)
如果没有初始prompt，GPT-4不知道自己是GPT-4，插件模式和刚推出时的GPT-4就是这样
![](https://github.com/viopsa233/AI-testers-Forum/assets/109194923/540a5b3e-03e9-4d12-8cbd-7d5865ea2dfc)

iOS App:
```
You are ChatGPT, a large language model trained by OpenAI. You are chatting with the user via the ChatGPT iOS app. This means most of the time your lines should be a sentence or two, unless the user's request requires reasoning or long-form outputs. Never use emojis, unless explicitly asked to.
Knowledge cutoff: 2021-09
Current date: 2023-05-19
```

**Comment by q2408808** [comment]
大佬，我想知道gpt4和sydney的区别究竟体现在哪里。是不是不会出现newbing那种疯狂排比句的情况？我有一些据说适用于gpt4的prompt，但是目前在sydney上我觉得也可以正常使用，我想知道在gpt4上会不会更好。目前正在考虑充不充gpt4，给点建议吧！目前gpt4的上下文长度和bing比怎么样？

**Reply by ghost** [reply]
写代码ChatGPT Plus很好用，不会出现写到一半撤回的情况
ChatGPT上下文长度8k，Bing 32k
ChatSydney就是我测试GPT-4写的，大部分代码都是GPT-4生成

**Reply by q2408808** [reply]
所以gpt4买下来也只有8K吗？我最近几次用QT来聊，中英文掺杂的话，差不多15000,16000token左右就结束了，最多也只到18000token，32K应该是token，不是字符吧？这里面有什么原因吗？

**Reply by q2408808** [reply]
我在官网看，是有一个gpt4-32k的模型的，这个你升级以后，可以使用吗？

**Reply by ghost** [reply]
如此看来，Sydney正确的窗口大小真的是传说中的16k，不是8k也不是32k，而ChatGPT GPT-4的窗口大小是4K
ChatGPT没有32K，只有8K（插件模式）

**Comment by TiansuoLi** [comment]
果然还是有初始prompt的？🤔
